src/components/ui/bullets.js

```diff
@@ -0,0 +1,13 @@
+import { BodyText } from '@/ui/typography';
+import { EllipseIcon } from '@/svgs/icons';
+
+export const Bullet = ({ className, children }) => {
```

Comment:
```suggestion
export const Bullet = ({ className='', children }) => {
```
This covers an issue where the className is undefined. Also, this component needs docstring.